tsl/src/bgw_policy/policies_v2.c

```diff
@@ -630,7 +630,8 @@ policies_show(PG_FUNCTION_ARGS)
 	funcctx = SRF_PERCALL_SETUP();
 	lc = (ListCell *) funcctx->user_fctx;
 
-	if (lc == NULL)
+	/* clang doesn't understand that jobs won't be NIL due to the above FIRSTCALL */
+	if (lc == NULL || jobs == NIL)
```

Comment:
Nit: maybe an assert is better to tell the compiler what is expected.

Reply:
Ok, used an assert here now.

Reply:
@mkindahl so, it's possible that `jobs` might be legitimately `NIL` if there's no job entry for the provided hypertable. So can't use an assert here. Will go back to my earlier solution